src/server.ts

```diff
@@ -85,6 +87,7 @@ export class HomebridgePluginUiServer {
           return this.sendResponse(request, { message: e.message, error: e.requestError }, false);
         } else {
           console.error(e);
+          // @ts-ignore
           return this.sendResponse(request, { message: e.message }, false);
```

Comment:
This one is similar, but `e` used to be assumed to be an `Error` (I think) and is now `unknown`. You can do an inline cast here if you are ok assume it's an `Error`

```suggestion
          return this.sendResponse(request, { message: (e as Error).message }, false);
```